CHARTER.md

```diff
@@ -0,0 +1,117 @@
+# Project Charter
+
+## Overview
+
+Crossplane is a framework for building cloud native control planes without
+needing to write code. It has a highly extensible backend that enables you to
+build a control plane that can orchestrate applications and infrastructure no
+matter where they run, and a highly configurable frontend that puts you in
+control of the schema of the declarative API it offers.
+
+## Control Planes
+
+Control planes first emerged in network packet switching, where devices must
+forward packets as fast as possible, but also be highly configurable. Breaking
+out the control plane from the data plane allows the former to be optimized for
+flexibility while the latter is optimized for speed. It also decouples the
+failure domains - a control plane failure does not prevent the data plane from
+functioning. Control planes are rooted in control theory, in particular closed
+loop control where the actual state of a system is observed in order to
+determine how best to drive it toward a desired state.
+
+Control planes power cloud computing. Cloud providers like AWS are built with
+control planes, as are projects like Kubernetes. In cloud computing the data
+plane is instead a “compute plane” consisting of applications and the
```

Comment:
The term "compute plane" has been somewhat divisive when I've tested it with folks so far. I'm not married to it, but I would love a succinct way to address the "other plane" - i.e. what Crossplane is a control plane _for_.

Two alternatives that I don't love are:

* Call it a "data plane". This is the typical term but seems odd outside of a network context.
* Use "infrastructure plane" or "application plane". I would prefer a term inclusive of both.

I'm open to suggestions here.

Reply:
How about "runtime plane"? It sounds a bit more application-focused, but it can also represent infrastructure or data. It also emphasises the fact that the control process and environment itself are dynamic.

Reply:
Maybe "resource plane"?  Since crossplane "controls" (or "manages") resources, both composite and "managed" or native?

Reply:
Thanks folks! I personally like "runtime plane" and "resource plane" about equal to "compute plane". I'll keep it open for suggestions and perhaps the other steering committee members can chime in.

Reply:
After discussion with the steering committee we're going to just stick with "compute plane".